.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -44,22 +44,25 @@ those (please be careful to remove any personal or private data). Please surroun
 <!-- IMPORTANT: please answer the following questions, to help us narrow down the problem -->
 
 <!-- Was this issue identified on matrix.org or another homeserver? -->
-- **Homeserver**: 
+- **Homeserver**:
 
 If not matrix.org:
 
 <!--
 What version of Synapse is running?
-You can find the Synapse version by inspecting the server headers (replace matrix.org with
-your own homeserver domain):
-$ curl -v https://matrix.org/_matrix/client/versions 2>&1 | grep "Server:"
+You can find the Synapse version with this command:
+
+$ curl http://localhost:8008/_synapse/admin/v1/server_version
+
+(You may need to replace `localhost:8008` if synapse is not configured to
+listen on that port.)
```

Comment:
```suggestion
listen on that port).
```

Reply:
If the whole sentence is inside the parens, shouldn't the full-stop be? :man_shrugging: